2-mandatory/5-writing-tests.js

```diff
@@ -35,41 +35,57 @@ function convertScoreToGrade(score) {
   passes.
 */
 test("a score of 83 is grade A", () => {
-  expect(convertScoreToGrade(83), "Z");
+  expect(convertScoreToGrade(83)).toEqual("A");
 });
 
 /*
   The rest of the tests have comments describing what to test and you need to
   write a matching test
 */
 
-test.skip("a score of 71 is grade B", () => {
+test("a score of 71 is grade B", () => {
+  expect(convertScoreToGrade(71)).toEqual("B");
   /* Remove the .skip above, then write the test body. */
 });
 /*
   Write a test that checks a score of 68 is grade C
 */
-
+test("a score of 68 is grade C", () => {
+  expect(convertScoreToGrade(68)).toEqual("C");
+});
 /*
   Write a test that checks a score of 55 is grade D
 */
-
+test("a score of 55 is grade D", () => {
+  expect(convertScoreToGrade(55)).toEqual("D");
+});
 /*
   Write a test that checks a score of 68 is grade C
 */
-
+test("a score of 68 is grade C", () => {
+  expect(convertScoreToGrade(68)).toEqual("C");
+});
 /*
   Write a test that checks a score of 55 is grade D
 */
-
+test("a score of 55 is grade D", () => {
+  expect(convertScoreToGrade(55)).toEqual("D");
+});
 /*
   Write a test that checks a score of 49 is grade E
 */
-
+test("a score of 49 is grade E", () => {
+  expect(convertScoreToGrade(49)).toEqual("E");
+});
 /*
   Write a test that checks a score of 30 is grade E
 */
-
+test("a score of 30 is grade E", () => {
+  expect(convertScoreToGrade(30)).toEqual("E");
+});
```

Comment:
your tests look good! Nice job 
--Could you think of other ways to write the tests?

Reply:
Yes! We can solve this using If/else statements, using an array to a function that converts score to grade, or using switch.